R/await.R

```diff
@@ -134,8 +134,9 @@ await_all <- function(f, .x, .y = NULL, ...,
   zipped_parameters <- if (is.null(.y)) .x else mapply(c, .x, .y, SIMPLIFY=FALSE)
 
   repeat {
-    responses[!called] <- lapply(zipped_parameters[!called], safe_call_once,
-                                 f = f, ..., .status_key = .status_key,
+    responses[!called] <- lapply(X = zipped_parameters[!called], FUN = safe_call_once,
```

Comment:
We may need a slightly different version of `safe_call_once` to get this to work.